loader/standard_loader.py

```diff
@@ -196,8 +196,10 @@ def _load_bundle_concurrent(self, count, parsed_bundle):
 
     def _load_parsed_bundles_concurrent(self):
         """Loads already parsed bundles concurrently using threads"""
-        patch_connection_pools(maxsize=256)
-        with concurrent.futures.ThreadPoolExecutor() as executor:
+        # This is now redundant but will be useful if we remove the artificial limit on the number
+        # of workers
+        # patch_connection_pools(maxsize=256)
```

Comment:
Hi Jesse, you have looked into this more than I have, yet I am inclined to think we still want/need the `patch_connection_pools`. The default value (if we don't patch it) is 1. We will have up to 5 worker threads, and each thread thread will hit at least 4 different HTTP(S) endpoints that I can think of, and maybe more.
We may probably won't need all 256, but I think the pool will only actually contain as many connections as the "high water mark" for the number of worker threads we are using. So, I think it is fine at 256, but we could make it a smaller number if you think that's appropriate/preferable (e.g. 64 or 128?).

Reply:
I overlooked this. I agree with you. I'll set it to something smaller.